core/utils/poly-look/src/visualisations/charts/bar-charts/horizontalBarChart/horizontalBarChart.js

```diff
@@ -237,13 +237,16 @@ export class HorizontalBarChart extends Chart {
       .attr("fill", this._barValueColor);
   }
 
-  _updateExistingBarValues(barValues) {
-    barValues
+  _updateExistingBarValues(barGroups) {
+    barGroups
+      .selectAll("bar-value")
```

Comment:
Here you're changing the code to select child elements of `barGroups` (previously `barValues`) -> Is that the fix?